tools/manifest-templator/manifest-templator.go

```diff
@@ -41,6 +41,11 @@ import (
 	"kubevirt.io/kubevirt/tools/util"
 )
 
+const (
+	customImageExample   = "Examples: some.registry.com@sha256:abcdefghijklmnop, other.registry.com:tag1"
```

Comment:
`abcdefghijklmnop` is not a valid example for sha digest as it should include only hexadecimal characters (0-9,a-f) :)